mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -574,8 +574,9 @@ def test_load_bad_channels(tmp_path):
 
     # Test forcing the bad case
     with pytest.warns(RuntimeWarning, match='1 bad channel'):
-        raw.load_bad_channels(bad_file_wrong, force=True)
-        # write it out, read it in, and check
+        raw.load_bad_channels(bad_file_wrong, force=True, verbose=None)
```

Comment:
verbose=None here is a noop, you shouldn't need to add this

Reply:
I figured it'd be a good smoke test to make sure it accepts that argument?

Reply:
If you really want to check functionally, you can make an identical call with verbose='error' without the warns context manager.

I don't think we need to start adding tests for verbose=None, we use it many places and I wouldn't want to see a bunch of verbose=One's added to our tests

Reply:
okay, I think you are right, I removed that in 04f2e07